README.md

```diff
@@ -143,4 +168,23 @@ If the Post creation is successful, the new Post should be printed to the browse
 
 ```js
 // Your answer goes here...
+$(document).ready(()=>{
+$(".post").on("click", (() => {
+    $.ajax({
+      url = "http://localhost:3000/posts"
```

Comment:
In JS, it's 'illegal' to assign a var inside of an object literal.
Should be...
```js
url: "http://localhost:3000/posts.json"
```